app/models/article.rb

```diff
@@ -555,7 +555,9 @@ def video_duration_in_minutes
   end
 
   def update_score
-    self.score = reactions.sum(:points) + Reaction.where(reactable_id: user_id, reactable_type: "User").sum(:points)
+    spam_adjustment = user.spam? ? -500 : 0
+    negative_reaction_adjustment = Reaction.where(reactable_id: user_id, reactable_type: "User").sum(:points)
```

Comment:
I don't think this line necessary means it's will be "negative" though, correct?

Reply:
_Technically_ it would always be negative but your right that this is kind of weird wording because that logic is elsewhere in the codebase. This line shouldn't know about it.